### PR TITLE
login modified

### LINKED ISSUE #43: make a login page in hotel-admin-frontend
**Is your feature request related to a problem? Please describe.**
make one login page inside ```hotel-admin-frontend/src/pages``` and name it ```Login.js``` use React Bootstrap forms
there should be the following fields

1.Username
2.Password
3.Captcha Validation
4. Proper validation check for Valid Email formats in the ```Username``` Field
5. If captcha is not done then form can't be submitted 
6. On Successful submission print the formdata in consul.log

### PR DESCRIPTION
**Pull request by Sapna2001**
Fixes #43 

#### Changes done

- Login page created

Screenshots of the changes (If any) -
Screenshot from 2021-03-24 00-52-08

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Review by ayan-biswas0412 — COMMENTED** [review]
in which route your page is ?

**Comment by Sapna2001** [comment]
@ayan-biswas0412 I had just created the Login.js file inside hotel-admin-frontend/src/pages as mentioned in the issue. I didn't create a route. Do you want me to add a route in the app.js file?

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412 I had just created the Login.js file inside hotel-admin-frontend/src/pages as mentioned in the issue. I didn't create a route. Do you want me to add a route in the app.js file?

yes

**Comment by Sapna2001** [comment]
> > @ayan-biswas0412 I had just created the Login.js file inside hotel-admin-frontend/src/pages as mentioned in the issue. I didn't create a route. Do you want me to add a route in the app.js file?
> 
> yes
@ayan-biswas0412  Route path = /login

**Review by ayan-biswas0412 — APPROVED** [review]
can you provide the credentials with which I can login ?

File: hotel-admin-frontend/src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          <Form.Control
+            name = "password"
+            type="password"
+            placeholder="password"
+            onChange = {this.handleChange}
+            value = {this.state.password}
+          />
+        </Form.Group>
+        <Recaptcha
+          sitekey = "6Lcl5nMaAAAAAHgjRvfoNs4mR5Qj0VIAHhtTgfZY"
```
[the remark this answers is not in the document]
Comment (Sapna2001) [reply hotel-admin-frontend/src/pages/Login.js:74]: @ayan-biswas0412 You can test it here: https://www.google.com/recaptcha/admin/site/443803173
I have given you access (ayanbiswas184@gmail.com)

Screenshot (58)

**Pull request closed without merging**